styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -331,17 +364,156 @@
 
 .caseStudy-text {
   font-family: $copy;
-  font-size: 1.75rem;
+  font-size: 1.25rem;
   letter-spacing: .5px;
-  line-height: 2.5rem;
+  line-height: 1.5rem;
+  word-spacing: .5px;
 }
 
 .caseStudy-button {
   border-radius: 3px;
-  text-transform: capitalize;
+  color: $rackspace-red;
+  cursor: pointer;
+  font-weight: 400;
+  text-decoration: none;
   width: 250px;
 }
+// < 480px
+@media only screen and (max-width: $screen-xs) {
+  .caseStudy-wrapper,
+  .caseStudy-main-image {
+    background-image: url('../imgs/derek/banners/customer-story-background-480.jpg');
+  }
+
+  .caseStudy-main-image {
+    display: block;
+    height: 240px;
+  }
+
+  .caseStudy-bar {
+    border-top: 1px solid $gray-light;
+    padding-top: 10px;
+  }
+
+  .container {
+    div {
+      :last-child {
+        .caseStudy-box {
```

Comment:
We shouldn't ever need to use element-based selectors, and there's a lot of nesting going on here. In these particular cases I'd just give each box a unique name. 
For example, instead of something like this: 
```
<div class="caseStudy-box col-md-3 col-md-offset-6 col-sm-6">
```

let's go with 
```
<div class="caseStudy-box caseStudy-box-story1">
```

Then we just define the css like: 

```
.casestudy-box-story1 {
  @include make-sm-column(6);
  @include make-md-column(3);
  @include make-md-column-offset(6);
}
```

This allows us to simplify the markup and get rid of some of the duplicative bootstrap selectors. 

We'd then need to do that for all of the other boxes as well.